.azure-pipelines/INSTALL.md

```diff
@@ -0,0 +1,112 @@
+# Configuring Azure Pipelines with Certbot
+
+Let's begin. All pipelines are defined in `.azure-pipelines`. Currently there are two:
+* `.azure-pipelines/main.yml` is the main one, executed on PRs for master, and pushes to master,
+* `.azure-pipelines/advanced.yml` add installer testing on top of the main pipeline, and is executed for `test-*` branches, release branches, and nightly run for master.
+
+Several templates are defined in `.azure-pipelines/templates`. These YAML files aggregate common jobs configuration that can be reused in several pipelines.
+
+Note that `CODECOV_TOKEN` secured environment variable needs to be set to allow the main pipeline to publish coverage reports to CodeCov.
```

Comment:
Yes, it is because of the partial support of Azure Pipeline on Codecov.

In fact there is three level of supports for a CI:
1) flawless support: in this case, you do not need to provide `CODECOV_TOKEN`, and reports integrates flawlessly in Codecov
2) partial support: the reports integrates flawlessly in Codecov if you provide the token through `CODECOV_TOKEN`
3) no support: you can still push coverage reports if you provide the token through `CODECOV_TOKEN`, but because of missing optional other parameters consumed from the CI environment, reports are not integrated nicely

At this point, `python-coverage` is in situation 3), and `coverage-bash` is in situation 2).

That is why I moved to `coverage-bash`. I update the comment here.